homeassistant/components/glances/sensor.py

```diff
@@ -344,6 +354,24 @@ def available(self) -> bool:
     @property
     def native_value(self) -> StateType:
         """Return the state of the resources."""
+        if self.entity_description.type == "uptime":
```

Comment:
Should not have these kind of "calculations" in the properties.
Why not just adding the starting time instead as I would understand is the base here?

Reply:
I believe glances sends uptime as a formatted string like "12 days, 3:42:29", not the original startup time, so converting to a timestamp needs caulations.
The other issue I see is that the calcualtion will not be very prcecise, so the computed Timestamp will change slightly at each update.
Maybe passing along the uptime as a string without computing a timestamp would be fine ?

Reply:
Ah. Didn't look that careful but just assumed.
We should provide it as a proper timestamp.

But the calculation should be in the `DataUpdateCoordinator` and not here (would also remove a few comments as below)

Reply:
Thanks, I will move the calculations, but as @wittypluck already mentioned glances doesn't provide a proper timestamp, but instead a human readable form.